README.md

```diff
@@ -1,31 +1,58 @@
+# SD-JWT
+
 This is an implementation of [SD-JWT (I-D version 05)](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-05.html) in typescript.
 
+## Functionalities
 
-### Test
+- Configurable:
+  - [ ] Hasher / Hashing ALG
+  - [ ] Signer
+  - [ ] Salt generator
 
-Runs against examples in `test/examples` directory
+- Issue SD-JWT:
+  - [x] Support recursive disclosures (parent object and its keys)
+  - [x] Support for nested objects
+  - [x] Support for arrays
+  - [x] Optional: public key binding (cnf)
+  - [ ] Optional: decoy digest
 
-Examples are generated using [`sd-jwt-generate`](https://github.com/openwallet-foundation-labs/sd-jwt-python)
-```
-npm run test
-```
+- Verify SD-JWT:
+  - [x] Support recursive disclosures (parent object and its keys)
+  - [x] Support for nested objects
+  - [x] Support for arrays
+  - [x] Optional: public key binding (cnf) check against the key binding JWT if one was provided
+  - [x] Optional: decoy digest
 
-# packSDJWT Examples
+- Additional:
+  - [ ] Holder: separate function for a holder to be able to create a key binding JWT (6.2.2)
+
+- Tests:
+  - [x] Issuer SD-JWT tests
+  - [x] Verifier SD-JWT tests
+  - [ ] Tests that check compatibility with SD-JWT generated by external libs
```

Comment:
There's tests that test the verification against the examples generated using the python lib.